content/blog/four-lenses-of-productivity.md

```diff
@@ -0,0 +1,88 @@
++++
+authors = ["Sapan Parikh"]
+date = 2023-07-24T00:00:00Z
+description = "How productivity varies at individual, teams, organization and market leve and what can impact it."
+draft = false
+image = "/images/2021/11/checklist2.jpg"
+slug = "four-lenses-of-productivity"
+tags = ["DevEx"]
+title = "Four Lenses of Productivity: Incubyte Take"
++++
+
+{{< circlepack data="ind-team-org-market.csv" >}}
```

Comment:
The data param says csv, but the file we have is a json: `ind-team-org-market.json`
Is that intentional?